src/state/lists/reducer.test.ts

```diff
@@ -1,6 +1,6 @@
 import { DEFAULT_ACTIVE_LIST_URLS } from './../../constants/lists'
 import { createStore, Store } from 'redux'
-import { DEFAULT_LIST_OF_LISTS } from 'constants/lists'
+import { DEFAULT_LIST_OF_LISTS } from '@src/constants/lists'
```

Comment:
Why do we import DEFAULT_LIST_OF_LISTS from '@src/constants/lists' and DEFAULT_ACTIVE_LIST_URLS from ./../../constants/lists? @W3stside 

I think we should use our custom defaults, right?

Reply:
why would we do this in their base test? we only import ours in the relevant reducer, which would be in custom.

Reply:
the import is weird but that's their original code. the escape hatch `@src` is because, as I've mentioned before, Uni is switching to absolute imports.

Reply:
okok, fair enough